### PR TITLE
#555 Add channel edition for the owner

### PR DESCRIPTION
**Pull request by stephanevieira75**

### CODE REVIEW

File: twake/frontend/src/app/scenes/Client/ChannelsBar/Parts/Channel/ChannelMenu.tsx [hunk 1]
```diff
@@ -185,6 +185,11 @@ export default (props: Props): JSX.Element => {
         },
       },
     );
+
+    // This remove the edit channel menu item when user isn't channel owner
+    if (props.channel.owner !== currentUser.id) {
+      menu.splice(4, 1);
```
Comment (chamerling) [inline twake/frontend/src/app/scenes/Client/ChannelsBar/Parts/Channel/ChannelMenu.tsx:191]: Is it the way to do in react, seriously? Can't we apply rules on each menu item based on the user roles?

Comment (stephanevieira75) [reply twake/frontend/src/app/scenes/Client/ChannelsBar/Parts/Channel/ChannelMenu.tsx:191]: Something like this ?

```js
{
      type: 'menu',
      rules: props.channel.owner === currentUser.id,
      text: Languages.t('scenes.app.channelsbar.modify_channel_menu'),
      onClick: () => {
        editChannel();
      },
}
...

const filteredMenu = menu.filter((item: any) => item.rules !== false);
```

Comment (RomaricMourgues) [reply twake/frontend/src/app/scenes/Client/ChannelsBar/Parts/Channel/ChannelMenu.tsx:191]: We could add if / else while pushing menu elements, but your proposition @stephanevieira75 is good, I really like it.
Can you:
- rename "rules" to "disabled" (and so inverse the condition)
- put this line 175 of MenusBodyLayer:
```
          {this.state.menus_manager.menus.filter(item=>!item.disabled).map((item, i) => {
```

Comment (chamerling) [reply twake/frontend/src/app/scenes/Client/ChannelsBar/Parts/Channel/ChannelMenu.tsx:191]: Well, item can be enabled/disabled/hidden, so is `disabled` the right name?

Comment (RomaricMourgues) [reply twake/frontend/src/app/scenes/Client/ChannelsBar/Parts/Channel/ChannelMenu.tsx]: We already had a "hide" option in the menu builder, so we used it 👍

**Pull request merged**